### ISSUE TITLE
Option to fail on "I don't have any idea what to do with" 

### ISSUE THREAD

**Issue by alimoeeny** [travis-ci/travis-ci#8298]
Hi there, I maybe missing something here, I apologize in advance.
I want the build to fail if there is a problem in my `travis.yml` file. 
Right now I have:

    language: go
    
    go:
        - 1.9.x

and in the build log I see:

    $ GIMME_OUTPUT=$(gimme 1.9.x | tee -a $HOME/.bashrc) && eval "$GIMME_OUTPUT"
    I don't have any idea what to do with '1.9.x'.
      (using type 'auto')

But it does not fail. 
Is there an option to make the build fail in these situations?
Shouldn't this be the default behavior?

**Comment by BanzaiMan** [comment]
Do you have a build log URL that shows the problem you are describing here?

**Comment by alimoeeny** [comment]
Hi @BanzaiMan sure, I have a public one on .org 
https://travis-ci.org/AdRoll/goamz/jobs/268421261
But also private ones on .com which don't use the "Precise environment" and they show the exact same lines in the log:

    $ GIMME_OUTPUT=$(gimme 1.9.x | tee -a $HOME/.bashrc) && eval "$GIMME_OUTPUT"
    I don't have any idea what to do with '1.9.x'.
      (using type 'auto')

**Comment by BanzaiMan** [comment]
Please try restarting. We had a release yesterday which is supposed to fix this issue.

**Comment by alimoeeny** [comment]
mmmm, doesn't look like it.
Neither it fails nor supports go 1.9.x .
I just restarted both the public one, link below and the private ones, same behavior as before.
here https://travis-ci.org/AdRoll/goamz/jobs/268421261

**Comment by BanzaiMan** [comment]
Oh. I realized that I had not deployed it. Will do that tomorrow.

**Comment by BanzaiMan** [comment]
Deployed it earlier this morning. Please restart the builds again and let us know how it goes.

**Comment by alimoeeny** [comment]
Thanks @BanzaiMan , I don't know if "that problem" has been solved, but now there is support for go 1.9 and everything works as expected. 
To make it clear, there were two separate issues:
1- 1.9.x was not being recognized, which now it is. thank you.
2- instead of throwing an error and failing, the unrecognized 1.9.x was just being ignored by the message/ warning : 'I don't have any idea what to do with ....' This second problem may or may not have been solved by your new code. 

Thanks, and as far as I can be involved this issue can be closed. I appreciate your help.

**Comment by BanzaiMan** [comment]
@alimoeeny Indeed, item 2 is not addressed. We'll take a look at that shortly.

**Comment by batazor** [comment]
I have this problem, as for example https://travis-ci.com/batazor/shortlink/builds/133099490

```
$ travis_setup_go
I don't have any idea what to do with '1.13.3'.
  (using download type 'auto')
Failed to run gimme
The command "travis_setup_go" failed and exited with 86 during .
```

@alimoeeny

**Comment by stefantalpalaru** [comment]
This problem resurfaced again: https://travis-ci.org/status-im/nim-beacon-chain/jobs/619477986#L143
